mmocr/models/textrecog/backbones/svtr.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Modified from <https://arxiv.org/abs/2205.00159>
+# Adapted from <https://github.com/PaddlePaddle/PaddleOCR>
+from typing import Callable
+
+import torch.nn as nn
+from mmengine.model import BaseModule
+
+
+class Conv_BN_Layer(BaseModule):
```

Comment:
We can use [ConvMoudle](https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/bricks/conv_module.py#L18) in mmcv instead

Reply:
Thanks for your reminding, I have used ConvModule instead.